client/components/auth/user.service.js

```diff
@@ -31,6 +31,13 @@ function UserResource($resource) {
         controller:'events'
       },
       isArray: true
+    },
+    getFullSections: {
+      method: 'GET',
+      params: {
+        controller:'instructor',
+        options: 'fullsections'
+      }
```

Comment:
you can't fix this because the changes are on another branch and not ready for PR @kmcnellis but keep in mind it will break